x/blob/types/params.go

```diff
@@ -84,13 +84,6 @@ func validateGovMaxSquareSize(v interface{}) error {
 		return fmt.Errorf("gov max square size cannot be zero")
 	}
 
-	if govMaxSquareSize > appconsts.MaxSquareSize {
-		return fmt.Errorf(
-			"gov max square size cannot exceed the max square size: max %d",
-			appconsts.MaxSquareSize,
-		)
-	}
-
```

Comment:
[question] out of curiosity why is this part of validation no longer necessary?

Reply:
This was discussed out of band. The crux is that validation of params does not accept any arguments yet this capped value depends on the version of the network, thus we can no longer correctly validate it here and need to do so elsewhere